packages/calling/src/CallingClient/calling/call.ts

```diff
@@ -2373,6 +2373,24 @@ export class Call extends Eventing<CallEventTypes> implements ICall {
     });
   }
 
+  /* istanbul ignore next */
+  /**
+   * Copy SDP's c-line to session level from media level.
+   * SPARK-522437
+   */
+  private addSessionConnection(sdp: string): string {
```

Comment:
Can we name it as updateSDP ?

Reply:
`updateSDP` - looks very generic. That is why I named it so.

Would something like `addSessionConnectionToSDP` be better?

Reply:
This code will be moved anyway so not a blocker